src/components/MessageList/MessageList.js

```diff
@@ -6,20 +6,18 @@ import { connect } from 'react-redux'
 import styles from './MessageList.scss'
 import Message from '../Message'
 
-function renderMessages(messages = []) {
+function renderMessages(messages = [], x, y) {
   return messages.map((elem, idx) => {
     return (
-      <Message
-        key={idx}
-        message={elem}/>
+      <Message key={elem.id} message={elem} x={x} y={y} />
```

Comment:
what i do usually in this case is to type each prop in a single line,
For example, 
<Component
  prop1 = val1
  prop2 = val2
  ...>